### DISCUSSION TITLE
Are there more easter eggs?

### DISCUSSION THREAD

**Discussion by ShizukoV** [monkeytypegame/monkeytype#1670] [Questions] [answered]
I mean besides power mode and monkey :P

**Comment by unrealapex** [comment, marked as answer]
Carrot and Banana carets ...

**Reply by ShizukoV** [reply]
Nice! Haven't found those yet :D

**Reply by unrealapex** [reply]
Also the monkey:
![Screenshot 2021-09-06 1 54 30 PM](https://user-images.githubusercontent.com/79121360/132260055-eeeeabb7-281f-4e2a-a84a-5a98e5aa5108.png)

**Reply by unrealapex** [reply]
There are probably more. You can find them in the [commandline-lists.js](https://github.com/Miodec/monkeytype/blob/master/src/js/commandline-lists.js) file.

**Reply by ShizukoV** [reply]
I already know the monkey one lol. I scrolled through and I don't think that there's more, but I figured out how to get the banana and carrot carets. :D
